debezium-core/src/main/java/io/debezium/jdbc/JdbcConnection.java

```diff
@@ -1458,4 +1458,22 @@ public PreparedStatement readTablePreparedStatement(CommonConnectorConfig connec
             throws SQLException {
         return rs.getObject(columnIndex);
     }
+
+    /**
+     * Converts a {@link ResultSet} row to an array of Objects
+     */
+    public <T extends DatabaseSchema<TableId>> Object[] rowToArray(Table table, T databaseSchema, ResultSet rs,
+                                                                   ColumnUtils.ColumnArray columnArray)
+            throws SQLException {
+        final Object[] row = new Object[columnArray.getGreatestColumnPosition()];
+        for (int i = 0; i < columnArray.getColumns().length; i++) {
+            row[columnArray.getColumns()[i].position() - 1] = getColumnValue(rs, i + 1,
+                    columnArray.getColumns()[i], table, databaseSchema);
+        }
+        return row;
+    }
+
+    public String quotedTableIdString(TableId tableId) {
+        return tableId.toDoubleQuotedString();
+    }
```

Comment:
Worth a JavaDoc comment given it's sort of an SPI.

Reply:
@gunnarmorling Added